cs/mailing.texy

```diff
@@ -179,7 +192,24 @@ $mailer->send($mail);
 \--
 
 
-Můžeme si také vytvořit vlastní mailer, jde o objekt implementující rozhraní [Nette\Mail\IMailer |api:].
+Fallback Mailer
+---------------
+
+Máme-li občas potíže s odesíláním emailů (např. výpadek smtp serveru), můžeme využít `FallbackMailer`, který umí v případě chyby automaticky zopakovat pokus o odeslání, buď pomocí stejného nebo pomocí alternativního maileru.
 
+`Fallback Mailer` nám dává několik funkcí navíc.
+
+- Využívat více mailerů.
+- Nastavit počet pokusů pro odeslání.
+- Nastavit čekací dobu mezi pokusy.
```

Comment:
To jsme tam puvodne meli, ale prislo mi to zbytecne. Resp, ta ukazka byla celkem dlooouha. Tak bych to zkusil dat do konstruktoru, jak rika @dg